src/webots/nodes/WbMuscle.cpp

```diff
@@ -46,18 +46,22 @@ const int SUBDIVISION = 16;            // ellipsoid subdivision
 WbVector2 *gCircleCoordinates = NULL;  // based on SUBDIVISION value
 
 void WbMuscle::init() {
-  mMaxRadius = findSFDouble("maxRadius");
+  mVolume = findSFDouble("volume");
   mStartOffset = findSFVector3("startOffset");
   mEndOffset = findSFVector3("endOffset");
   mColors = findMFColor("color");
   mCastShadows = findSFBool("castShadows");
   mVisible = findSFBool("visible");
 
+  // deprected field
+  mMaxRadius = findSFDouble("maxRadius");
+  if (mVolume->value() == 0.01 && mMaxRadius->value() != 0.0)
+    warn(tr("'maxRadius' field replaced by the 'volume' field, use this new field instead."));
```

Comment:
For consistency with other deprecation messages something like the following would be better:

```suggestion
    warn(tr("'maxRadius' field is deprecated, please use the 'volume' field instead."));
```

```suggestion
    warn(tr("Deprecated 'maxRadius' field, please use the 'volume' field instead."));
```